public/index.html

```diff
@@ -5,7 +5,9 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon_32.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="OwnPath by Colorado Behavioral Health Administration" />
+    <meta name="description" content="Find behavioral health care that’s meaningful for you in Colorado." />
+    <meta name="og:description" content="Find behavioral health care that’s meaningful for you in Colorado." />
+    <meta name=""og:image" content="https://ownpath.co/static/media/ownpath.8741e683b152244187c25936b930f9c8.svg" />
```

Comment:
alternatively you can pop the image in this `public` folder and use the `%PUBLIC_URL%` template var seen elsewhere in this file a la: `%PUBLIC_URL%/whateveryoucalltheimage.svg` but not necessary if this works!

Reply:
That'd be great! I'll reach out to see how I can set and use this during builds. It seems like in our current builds it's not set and just resolves to an empty string, and seems that sites like Facebook require an absolute URL rather than a relative one.